### ISSUE TITLE
CustomizedFabric causing issue for redux store

### ISSUE THREAD

**Issue by gbminnock** [OfficeDev/office-ui-fabric-react#6794]
Hi, 

I am adding redux to my stack but am finding problems with panels in that they say that they are rendered outside of root component. When I examine I see that the particular component is outside of the current tree usually in a 
`<CustomizedFabric className="ms-Layer-content cont-97>...</CustomizedFabric> `
component. 

Is there a workaround for this? 

possible a duplicate of: #5123

tks.

**Comment by cliffkoh** [comment]
What version of Fabric are you using? Latest Fabric Panel uses React Portals.

**Comment by gbminnock** [comment]
@cliffkoh 
I am using spfx version 1.5.1 (seems to be not released that long). I believe Portals is not supported. 

Anyway the versions I have are as follows: 

- @microsoft/sp-core-library@1.5.1
- @microsoft/sp-lodash-subset@1.5.1
- @microsoft/sp-office-ui-fabric-core@1.5.1
- @microsoft/sp-webpart-base@1.5.1
- office-ui-fabric-react-bundle@1.5.1
- office-ui-fabric-react@5.83.0
- office-ui-fabric-core@9.1.0
- office-ui-fabric@2.6.1
- react@15.6.2
- react-dom@15.6.2

I think I heard it good practice to stay with the yeoman generator version of things.

**Comment by cliffkoh** [comment]
Yes, you would have to stay with Fabric 5 as SPFX does not support Fabric 6 yet. 

Unfortunately, I'm not sure what we can do here in Fabric. Portals is a React 16 feature and there is no way for us to backport this to Fabric 5. The only way would be for SPFX to upgrade to React 16 and adopt/unblock the adoption of Fabric 6.

**Comment by gbminnock** [comment]
@cliffkof I managed to get it to work by passing the store down via props and then using "ownProps" in mapStateToProps. Bit messy but will do until SPFx upgrade. 
Tks.
